content/desktop/install/windows-install.md

```diff
@@ -93,6 +90,20 @@ For more information on setting up WSL 2 with Docker Desktop, see [WSL](../wsl/_
 >
 > Docker only supports Docker Desktop on Windows for those versions of Windows that are still within [Microsoft’s servicing timeline](https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet). Docker Desktop is not supported on server versions of Windows, such as Windows Server 2019 or Windows Server 2022. For more information on how to run containers on Windows Server, see [Microsoft's official documentation](https://learn.microsoft.com/virtualization/windowscontainers/quick-start/set-up-environment).
 
+{{< /tab >}}
+{{< tab name="Windows on Arm (Beta)" >}}
```

Comment:
Should this be:

```suggestion
{{< tab name="WSL 2 backend, ARM (Beta)" >}}
```

For the other tabs we parenthesize the arch though, so parenthesizing "Beta" here instead of ARM feels inconsistent. Maybe we should remove the parens for the arch on all tabs.

Reply:
Also I'm assuming this is for WSL2 since Hyper-V and WCOW is not supported. So should this section also include WSL prerequisites? And how about hardware requirements?

Reply:
> So should this section also include WSL prerequisites?

Yes, good catch.  They're the same as the Intel ones.
- WSL version 1.1.3.0 or later.
- Turn on the WSL 2 feature on Windows. For detailed instructions, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/windows/wsl/install-win10).

> And how about hardware requirements?

- 64-bit processor
- 8GB system RAM
- Enable hardware virtualization in BIOS. For more information, see [Virtualization](https://deploy-preview-20117--docsdocker.netlify.app/desktop/troubleshoot/topics/#virtualization).